protocol/rpcprovider/rpcprovider.go

```diff
@@ -87,7 +87,7 @@ func (rpcp *RPCProvider) Start(ctx context.Context, txFactory tx.Factory, client
 	rpcp.providerStateTracker = providerStateTracker
 	providerStateTracker.RegisterForUpdates(ctx, statetracker.NewMetricsUpdater(providerMetricsManager))
 	// single reward server
-	rewardServer := rewardserver.NewRewardServer(providerStateTracker, providerMetricsManager)
+	rewardServer := rewardserver.NewRewardServer(providerStateTracker, providerMetricsManager, rewardserver.NewRewardStore(rewardserver.NewLocalDB(".storage/rewardserver/badger")))
```

Comment:
Let's make this the default value but add a viper option to the config file when setting up the rpcprovider so this path could be modifiable, you can see similar examples on rpcprovider.go (i.e viper.GetString(metrics.MetricsListenFlagName))